docs/static/setup.sh

```diff
@@ -29,6 +39,30 @@ if [ ! -w `pwd` ]; then
     exit 1
 fi
 
+if [[ $curl_exists == "true" && $wget_exists == "true" ]]; then
+    prompt="Detected both curl and wget, which one would you like to use? [default is curl]"
+    options=("curl" "wget")
+    PS3="$prompt "
```

Comment:
I do not think prompting user will add any value in selecting one over the other, since most (regular) users does not care about it. Can proceed with whatever is available. 
But it might be worth to add an option to force using one over the other in some specific scenario.
Thoughts ?

Reply:
I like the force option, that will allow the users who do care the ability to choose.